### PR TITLE
HDDS-11320. Update OM, SCM, Datanode conf for RATIS-2135.

### PR DESCRIPTION
**Pull request by szetszwo**
## What changes were proposed in this pull request?

Due to a bug, [RATIS-2135](https://issues.apache.org/jira/browse/RATIS-2135) requires
- raft.grpc.message.size.max

to be 1MB larger than

- raft.server.log.appender.buffer.byte-limit

We must change the OM, SCM and Datanode conf accordingly.

## What is the link to the Apache JIRA

HDDS-11320

## How was this patch tested?

A new unit test.

### CODE REVIEW

**Review by jojochuang — APPROVED** [review]
LGTM. Thanks @szetszwo

File: hadoop-hdds/common/src/test/java/org/apache/hadoop/hdds/conf/TestRatisConfUtils.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    final int logAppenderBufferByteLimit = 1000;
+
+    // setMessageSizeMax without setBufferByteLimit
+    Assertions.assertThrows(IllegalStateException.class,
+        () -> RatisConfUtils.Grpc.setMessageSizeMax(properties, logAppenderBufferByteLimit));
+
+    RaftServerConfigKeys.Log.Appender.setBufferByteLimit(properties, SizeInBytes.valueOf(logAppenderBufferByteLimit));
+
+    // setMessageSizeMax with a different logAppenderBufferByteLimit
+    Assertions.assertThrows(IllegalStateException.class,
```
Comment (jojochuang) [inline hadoop-hdds/common/src/test/java/org/apache/hadoop/hdds/conf/TestRatisConfUtils.java:48]: but this API does not throw exception.

Comment (szetszwo) [reply hadoop-hdds/common/src/test/java/org/apache/hadoop/hdds/conf/TestRatisConfUtils.java:48]: Oops, I need to update the test.

**Comment by jojochuang** [comment]
LGTM Merging it.

**Comment by szetszwo** [comment]
@jojochuang , thanks a lot for reviewing and merging this!

**Pull request merged**